docs/querying/aggregations.md

```diff
@@ -430,17 +579,30 @@ A filtered aggregator wraps any given aggregator, but only aggregates the values
 
 This makes it possible to compute the results of a filtered and an unfiltered aggregation simultaneously, without having to issue multiple queries, and use both results as part of post-aggregations.
 
-*Note:* If only the filtered results are required, consider putting the filter on the query itself, which will be much faster since it does not require scanning all the data.
+If only the filtered results are required, consider putting the filter on the query itself, which will be much faster since it does not require scanning all the data.
```

Comment:
```suggestion
If only the filtered results are required, consider putting the filter on the query itself. This will be much faster because it does not require scanning all the data.
```